lib/sidekiq/cron/job.rb

```diff
@@ -197,7 +197,7 @@ def self.load_from_hash! hash
       def self.load_from_array array
         errors = {}
         array.each do |job_data|
-          job = new(job_data)
+          job = new(job_data.merge(source: "schedule"))
```

Comment:
I had 2 options, `regular` and `schedule`. I chose `schedule` because that goes well with `source`.